doc/manuals/user/transient_entities.md

```diff
@@ -0,0 +1,124 @@
+# Transient Entities
+
+Orion Context Broker implements *transient entities*. A transient entity is like any other entity 
+(i.e. it has id/type, a set of attributes, etc.) but with a limited lifetime, so when a given
+time has come, the entity is automatically deleted from the context managed by Orion.
+
+Thus, a first and very important piece of advice: **be careful if you use transient entities as once
+the expiration time has come, the entity will be automatically deleted from database and there is
+no way of recovering it**. Ensure the information you set in a transient entity is actually "transient"
+(i.e. it is not important after the lifetime of the entity has passed and it gets expired).
+
+In addition, **have a look to the [backward compatibility considerations section](#backward-compatibility-considerations)
+in the case you are already using attributes with the exact name `dateExpires`**.
+
+## The `dateExpires` attribute
+
+Transient entities are based in the `dateExpires` attribute. This is an attribute of `DateTime` type,
```

Comment:
The expiration timestamp of an Entity is defined by means of the `dateExpires` attribute

Reply:
Fixed in 78286a073f3b775bed26ebf74bb17da2bea33fe9